Podcast/FeedElement.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  FeedElement.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 9/27/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+enum FeedContext: String {
+    case followingRecommendation = "FOLLOWING_RECOMMENDATION"
```

Comment:
hmm thinking these could be more descriptive ... such as:
episodeRecommendationFromUser
seriesRecommendationFromUser

Reply:
I think these are actually more descriptive, if we decide to add recommendations from really popular users that you might not follow then that would also be an episodeRecommendationFromUser but not a followingRecommendation. I'll change it if you'd still prefer your suggestions though